composer.json

```diff
@@ -207,30 +207,30 @@
         ],
         "test": [
             "Composer\\Config::disableProcessTimeout",
-            "bin/runtests --all"
+            "@php bin/runtests --all"
         ],
         "phpstan": [
-            "bin/adminconsole cache:warmup --env=dev",
-            "vendor/bin/phpstan analyse"
+            "@php bin/adminconsole cache:warmup --env=dev",
+            "@php vendor/bin/phpstan analyse"
         ],
-        "php-cs": "vendor/bin/php-cs-fixer fix --verbose --diff --dry-run",
-        "php-cs-fix": "vendor/bin/php-cs-fixer fix",
+        "php-cs": "@php vendor/bin/php-cs-fixer fix --verbose --diff --dry-run",
+        "php-cs-fix": "@php vendor/bin/php-cs-fixer fix",
         "lint-composer": "@composer validate --strict",
-        "lint-twig": "bin/adminconsole lint:twig templates",
-        "lint-yaml": "bin/adminconsole lint:yaml config",
+        "lint-twig": "@php bin/adminconsole lint:twig templates",
+        "lint-yaml": "@php bin/adminconsole lint:yaml config",
         "lint-container": [
-            "bin/adminconsole lint:container --env dev",
-            "bin/websiteconsole lint:container --env dev",
-            "bin/adminconsole lint:container --env test",
-            "bin/websiteconsole lint:container --env test",
-            "bin/adminconsole lint:container --env stage",
-            "bin/websiteconsole lint:container --env stage",
-            "bin/adminconsole lint:container --env prod",
-            "bin/websiteconsole lint:container --env prod"
+            "@php bin/adminconsole lint:container --env dev",
+            "@php bin/websiteconsole lint:container --env dev",
+            "@php bin/adminconsole lint:container --env test",
+            "@php bin/websiteconsole lint:container --env test",
+            "@php bin/adminconsole lint:container --env stage",
+            "@php bin/websiteconsole lint:container --env stage",
+            "@php bin/adminconsole lint:container --env prod",
+            "@php bin/websiteconsole lint:container --env prod"
         ],
         "lint-doctrine": [
-            "bin/adminconsole doctrine:schema:validate --skip-sync",
-            "bin/adminconsole doctrine:ensure-production-settings --env prod"
+            "@php bin/adminconsole doctrine:schema:validate --skip-sync",
+            "@php bin/adminconsole doctrine:ensure-production-settings --env prod"
```

Comment:
Think this is good as `@php` would also use the same php version which composer was run e.g.:

```
php7.3 composer.phar lint-doctrine
```

will use php7.3 and not then the global php version.

👍 for this change, we should make this consistent in skeleton the bundles.

Reply:
sounds good! anybody interested in updating this in all bundles? 😅

Reply:
not really 🙈 The next one who does something in a bundle can just change this as well

Reply:
/cc @sulu/core-team